app/helpers/responses_helper.rb

```diff
@@ -26,6 +26,7 @@ def format_responses_field(resp, field)
       when "incomplete" then tbool(resp.incomplete?)
       when "reviewed" then reviewed_status(resp)
       when "user_id" then can?(:read, resp.user) ? link_to(resp.user.name, resp.user) : resp.user.name
+      when "reviewer_id" then resp.reviewer.present? ? (can?(:read, resp.reviewer) ? link_to(resp.reviewer.name, resp.reviewer) : resp.reviewer.name) : ""
```

Comment:
I think we should maybe split this out so that it's not one line -- chaining the ternary operator like this can be a little bit confusing. The parenthesis help but I think it'd be better for this to take up a couple of lines.